{ env: 'dev',
  port: 8080,
  jwt_secret: 'sldgkhoiu58340th0089y',
  database_uri: 'mongodb://user1:mlabkarma321@ds133533.mlab.com:33533/karma',
  logging_level: 'debug',
  authentication_timeout: 36000,
  portal_cache_timeout: 60,
  email: { title: 'Test mail from yoga', to: 'ssjkamil@gmail.com' },
  email_server: { username: 'example@gmail.com', password: 'password' },
  credentials: { username: 'admin', password: 'admin' } }
{ key:
   <Buffer 2d 2d 2d 2d 2d 42 45 47 49 4e 20 52 53 41 20 50 52 49 56 41 54 45 20 4b 45 59 2d 2d 2d 2d 2d 0d 0a 4d 49 49 45 6f 77 49 42 41 41 4b 43 41 51 45 41 78 ... >,
  cert:
   <Buffer 2d 2d 2d 2d 2d 42 45 47 49 4e 20 43 45 52 54 49 46 49 43 41 54 45 2d 2d 2d 2d 2d 0d 0a 4d 49 49 47 52 44 43 43 42 53 79 67 41 77 49 42 41 67 49 51 65 ... > }
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Object.<anonymous> (/root/server/server.js:86:8)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
